plugins/vbe/README.md

```diff
@@ -294,16 +273,6 @@ To build the example:
    ./build.sh
 ```
 
-To install the cab on your development computer:
-
-```bash
-   # Set up the test file
-   dd if=/dev/zero of=/tmp/testfw bs=1M count=3
-
-   sudo build/src/fwupdtool install plugins/vbe/example/Vbe-Board-1.2.4.cab \
-      bb3b05a8-ebef-11ec-be98-d3a15278be95
```

Comment:
Why drop this? It is hard to know how to test it otherwise.

Having said that, I think I need to create a proper test at some point.

Reply:
In general, only plugin-specific stuff goes in each plugin README.d, and the general stuff can go in the toplevel; in this case it's no different to any of the other plugins.

Reply:
What about the /tmp/testfw thing? I don't think others do something like that.

Anyway, this info is useful for me when I come back to it in a few months. Is there another place it could go?